scripts/publish-plugin.sh

```diff
@@ -1,46 +0,0 @@
-#!/bin/bash
```

Comment:
Why are we removing these scripts?

Reply:
I removed them because of their lack of generic usage. I've added them back as we'll undoubtedly need them until we have something better.